src/test/daml/Daml/Finance/Data/Test/DateClock.daml

```diff
@@ -0,0 +1,63 @@
+-- Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module Daml.Finance.Data.Test.DateClock where
+
+import Daml.Finance.Interface.Data.Reference.Time qualified as Time (Advance(..), I, Rewind(..))
+import Daml.Finance.Interface.Types.Common.Types (Id(..))
+
+import Daml.Finance.Data.Time.DateClock qualified as DateClock (DateClock(..), T)
```

Comment:
CI will probably fail as imports are not in alphabetical order (DA.* comes before Daml.*)